src/containers/Pembelian/Pembelian.tsx

```diff
@@ -108,7 +110,7 @@ export const DataPembelian = () => {
                         <td>{d.supplier}</td>
                         <td>{d.barang}</td>
                         <td>{d.biaya}</td>
-                        {cookies.user && (
+                        { user && (
```

Comment:
even not login, user still true because the default value of user is :
```
{
    id: '',
    userName: '',
    role: '',
    exp: 0,
    iat: 0,
  },
```

either make check the id if empty or make the user default value is null or undefined

Reply:
done set default values for user as undefined